packages/sdk-browser/src/__types__/base.ts

```diff
@@ -44,26 +53,23 @@ export class TurnkeyRequestError extends Error {
 
 export interface ActivityResponse {
   activity: {
-    id: string;
-    status: string;
+    id: TActivityId;
+    status: TActivityStatus;
     result: Record<string, any>;
   };
 }
 
 export interface ActivityMetadata {
   activity: {
-    id: string;
-    status: string;
+    id: TActivityId;
+    status: TActivityStatus;
   };
 }
 
 interface BaseSDKClientConfig {
   apiBaseUrl: string;
   organizationId: string;
-  activityPoller?: {
-    duration: number;
-    timeout: number;
-  };
+  activityPoller?: TActivityPollerConfig | undefined;
```

Comment:
Should this be considered a breaking change or nah?

Reply:
It could be, but at the same time don't feel like it quite justifies a major version bump. I've explicitly called it out in the changeset!

Reply:
Cool! I'm good with that 👍